components/Base/CounterButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <button
+    class="h-6 w-6 rounded-full flex items-center justify-center bg-gray-200"
+    @click="handleClick"
+  >
+    <PlusIcon v-if="isPlusButton" />
+    <MinusIcon v-else />
+  </button>
+</template>
+
+<script setup lang="ts">
+import PlusIcon from '~/assets/icons/plus.svg?skipsvgo';
+import MinusIcon from '~/assets/icons/minus.svg?skipsvgo';
+
+const props = defineProps<{
+  counter: number;
+  type: 'plus' | 'minus';
+}>();
+
+const emit = defineEmits(['update:counter']);
+
+const isPlusButton = computed(() => props.type === 'plus');
+
+const handleClick = () => {
+  isPlusButton.value ? incrementCounter() : decrementCounter();
+};
+
+const incrementCounter = () => emit('update:counter', props.counter + 1);
+const decrementCounter = () =>
+  emit('update:counter', props.counter > 0 ? props.counter - 1 : props.counter);
```

Comment:
`>= 0`, we can set 0 days

![image](https://user-images.githubusercontent.com/103642381/224854331-84f6556f-4b5a-4721-be91-12271b09c913.png)